src/api/exporters/exportJson.js

```diff
@@ -0,0 +1,59 @@
+import ezs from 'ezs';
+import ezsBasics from 'ezs-basics';
+import ezsLocals from '../statements';
+
+ezs.use(ezsBasics);
+ezs.use(ezsLocals);
+
+const exporter = (config, fields, characteristics, stream) => {
+    const getLabel = fields.reduce(
+        (data, field) => {
+            data[field.name] = field.label;
+
+            return data;
+        },
+        { uri: 'uri' },
+    );
+
+    const getLang = fields.reduce(
+        (data, field) => {
+            data[field.name] = field.language;
+
+            return data;
+        },
+        { uri: 'uri' },
+    );
+
+    return stream
+        .pipe(ezs('filterVersions'))
+        .pipe(ezs('filterContributions', { fields }))
+        .pipe(
+            ezs((resource, output) => {
```

Comment:
Don't use a inline function. It's should be better to create a standalone function in the ./statements/ directory. You'll be able to write a unit test, and to externalize the function.